frontend/src/store/slices/accessRequests/accessRequestsActionCreator.spec.tsx

```diff
@@ -0,0 +1,305 @@
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import * as genericActions from 'actions/genericActions';
+import * as API from 'constants/API';
+import * as MOCK from 'mocks/dataMocks';
+import { ENVIRONMENT } from 'constants/environment';
+import { useAccessRequests } from './useAccessRequests';
+import { renderHook } from '@testing-library/react-hooks';
+import configureMockStore, { MockStoreEnhanced } from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+import React from 'react';
+import { mockAccessRequest } from 'mocks/filterDataMock';
+
+const requestSpy = jest.spyOn(genericActions, 'request');
+const successSpy = jest.spyOn(genericActions, 'success');
+const errorSpy = jest.spyOn(genericActions, 'error');
+const mockAxios = new MockAdapter(axios);
+
+afterEach(() => {
+  mockAxios.reset();
+  requestSpy.mockClear();
+  successSpy.mockClear();
+  errorSpy.mockClear();
+});
+let currentStore: MockStoreEnhanced<any, {}>;
+const mockStore = configureMockStore([thunk]);
+const getStore = (values?: any) => {
+  currentStore = mockStore(values ?? {});
+  return currentStore;
+};
+const getWrapper = (store: any) => ({ children }: any) => (
+  <Provider store={store}>{children}</Provider>
+);
+
+describe('useAccessRequests functionality', () => {
```

Comment:
update the action creators test to consume the actions when formatted as a hook. Also increase the resolution of test expectations by checking the result of the action being executed against the store, rather than just the number of dispatch calls.